bin/fissile

```diff
@@ -13,14 +13,21 @@ if [ -f ${FISSILE_GIT_ROOT}/Jenkinsfile ] ; then
     # add the common part, which references them.
 
     if [ "${USE_SLE_BASE}" = "false" ]; then
-        SRC_BASE="${FISSILE_LIGHT_OPEN42}"
+        STACK_BASE="${FISSILE_LIGHT_OPEN42}"
     else
-        SRC_BASE="${FISSILE_LIGHT_SLE12}"
+        STACK_BASE="${FISSILE_LIGHT_SLE12}"
     fi
 
     tmp_file=$(mktemp -t tmp.XXXXXXXXXXXXXXXXXXXX)
```

Comment:
Huh we never bothered deleting the temp file if things fail did we?  Should probably fix that while we're here. (The usual `trap … EXIT` is fine)

Reply:
Like 
```
trap "rm -f $tmp_file" EXIT
```

Reply:
ok. waiting actually, and me redoing images etc to check the SSG manifest thing Jan mentioned